src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -150,7 +150,8 @@ private void createAccountPopup() {
      */
     private void deleteAccount() {
         if (chooseAccount.getSelectedAccount() == null) {
-            setupErrorPopup("Cannot delete the All Accounts aggregation", new Exception());
+            setupErrorPopup("Cannot delete the All Accounts aggregation. Please select an account " +
+                    "to delete.", new Exception());
```

Comment:
What is the point of throwing a blank exception? Why not have an overloaded method for setupErrorPopup that does not require an exception?